### ISSUE TITLE
New security measure preventing IP return

### ISSUE THREAD

**Issue by Justinzobel** [mpolden/echoip#163]
When using via curl I got `ifconfig.co needs to review the security of your connection before proceeding.` among other HTML returned instead of my IP.

**Comment by shawnflanagan** [comment]
I am having the same issue. Looks like Cloudflare is causing it. If I call [https://ifconfig.co/country-iso](https://ifconfig.co/country-iso) from my site hosted on Netlify, instead of returning the country ISO value it returns this HTML code:
<img width="517" alt="Screen Shot 2022-11-30 at 2 10 19 PM" src="https://user-images.githubusercontent.com/6181073/204898307-3eab7cb4-67f0-4ce4-9d14-08b38c5121c0.png">



It was first noticed on one of my sites around 17 November 2022.

**Comment by cyounkins** [comment]
I'm seeing this as well. I want to add that on my network there is only one service polling for IP changes, once every two minutes. This is within the requested limit of 1 request per minute described on the website.

**Comment by bennettp123** [comment]
I just noticed the same issue on an EC2 instance. I suspect this service is probably provided for free, so I can't imagine there's be much appetite for disabling captcha.

> Alternative: `dig +short txt ch whoami.cloudflare @1.0.0.1`

Nice one, thanks!

Another alternative (using https, in case port 53 is blocked):

```json
curl api.ipify.org
```

**Comment by hydrargyrum** [comment]
Yes, ifconfig.co is unusable via curl, but `api.ipify.org` works fine!

**Comment by TheKigen** [comment]
It is just a security setting that needs to be changed on the CloudFlare account running ifconfig.co.  I setup ipcfg.co as another.  Though other than the front end its entirely rewritten to just be a CloudFlare Worker.

**Comment by ifconfigla** [comment]
Another alternative for curl and tor users

```
curl ifconfig.la
```
